ice-hockey-manager-api/pom.xml

```diff
@@ -16,6 +16,11 @@
             <groupId>org.springframework.security</groupId>
             <artifactId>spring-security-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+            <version>1.0.0.GA</version>
```

Comment:
Each version should be listed in parent pom in multi-maven project, so that we avoid anomalies (e.g. different versions of dependency in different modules). I will fix it for now, later please specify any version there.